src/modules/video_player/index.js

```diff
@@ -6,6 +6,18 @@ const twitch = require('../../utils/twitch');
 
 const VIDEO_PLAYER_SELECTOR = '.video-player .player';
 
+let hidden, visibilityChange;
```

Comment:
none of this is necessary. `document.hidden` and `visibilitychange` are sufficient. we don't care about old browsers.